File: src/ssr_pkg/src/yao_node.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>
int main(int argc, char *argv[])
{
    ros::init(argc,argv,"yao_node");
    ros::NodeHandle nh;
    ros::Publisher chao_pub = nh.advertise<std_msgs::String>("/daifeiqun",1);
    std_msgs::String msg;
    msg.data = "求上车";
    ros::Rate loop_rate(10);

    while (ros::ok())
    {
        chao_pub.publish(msg);
        loop_rate.sleep();
    }
    return 0;
}

File: src/ssr_pkg/scripts/yao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

def main():
    rospy.init_node("yao_node_py")
    pub = rospy.Publisher("daifeiqunpy",String,queue_size=1)
    rate = rospy.Rate(10)
    while not rospy.is_shutdown():
        msg = String()
        msg.data = "求带飞py"
        pub.publish(msg)
        rate.sleep()
if __name__ == '__main__':
    main()
    